### ISSUE TITLE
Ringing after logout

### ISSUE THREAD

**Issue by moshibilpk** [react-native-webrtc/react-native-voip-push-notification#99]
The app is ringing after logout 
![image](https://user-images.githubusercontent.com/49393740/206232888-eed26633-31a3-4689-a511-2fadb9a197a1.png)

Is there any way to stop ringing after logout? like something to unregister from app

**Comment by nagyszili** [comment]
I think you should call `VoipPushNotification.removeEventListener('notification')` in your logout function

**Comment by moshibilpk** [comment]
> I think you should call `VoipPushNotification.removeEventListener('notification')` in your logout function

Yes, I tried, but it doesn't work for me. I believe the problem is with the call keep library because I use it.

**Comment by wilmxre** [comment]
@moshibilpk hey man, did you find a way to solve this?
